pkg/master/master.go

```diff
@@ -897,6 +898,10 @@ func (m *Master) getExtensionResources(c *Config) map[string]rest.Storage {
 	if c.APIResourceConfigSource.ResourceEnabled(version.WithResource("networkpolicies")) {
 		storage["networkpolicies"] = networkPolicyStorage
 	}
+	storageClassStorage := storageclassetcd.NewREST(restOptions("storageclasses"))
```

Comment:
We have testing in place and i'm not sure how we would enable or disable those specific tests with flag.

May we call this beta and enable by default?